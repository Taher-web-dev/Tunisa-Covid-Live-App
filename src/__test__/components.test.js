import React from "react";
import renderer from 'react-test-renderer';
import { render } from "@testing-library/react";
import NavBar from '../Navbar';
import { BrowserRouter as Router } from 'react-router-dom';
test('render correctly navbar', () => {
  const tree = renderer.create(
    <Router>
      <NavBar />
    </Router>
    ).toJSON();
  expect(tree).toMatchSnapshot();
});