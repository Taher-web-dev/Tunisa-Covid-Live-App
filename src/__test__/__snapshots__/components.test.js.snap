// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Homepage component should render with given state from redux store 1`] = `
<div
  className="home-page"
>
  <div
    className="title-board"
  >
    <div
      className="map"
    >
      <img
        alt="tunisia-map"
        src="map.png"
        style={
          Object {
            "height": "80%",
            "width": "25%",
          }
        }
      />
    </div>
    <p
      className="title-indicator"
    >
      Tunisia
       
      <span
        style={
          Object {
            "fontSize": "16px",
            "fontWeight": "normal",
          }
        }
      >
         
        +
        1.2
        % confirmed cases
         
      </span>
    </p>
  </div>
  <p
    className="sub-title"
    style={
      Object {
        "backgroundColor": "rgb(226, 77, 120)",
        "fontSize": "18px",
        "margin": "0 ",
        "padding": "2.5% 0 2.5% 2.5%",
      }
    }
  >
    KEY INDICATORS
  </p>
  <div
    className="key-indicators"
  >
    <a
      aria-current={null}
      className=" card confirmed-case"
      href="/ConfirmedCases"
      onClick={[Function]}
    >
      <img
        alt="back icone"
        src="fleche.png"
        style={
          Object {
            "margin": "2.5% 5% 0 0",
          }
        }
      />
      <p
        style={
          Object {
            "display": "flex",
            "flexDirection": "column",
            "fontSize": "18px",
            "fontWeight": "700",
            "marginRight": "5%",
            "textAlign": "end",
            "width": "50%",
          }
        }
      >
        NEW CASES
        <span>
          20
        </span>
      </p>
    </a>
    <div
      className="card deaths"
    >
      <img
        alt="back icone"
        src="fleche.png"
        style={
          Object {
            "margin": "2.5% 5% 0 0",
          }
        }
      />
      <p
        style={
          Object {
            "display": "flex",
            "flexDirection": "column",
            "fontSize": "18px",
            "fontWeight": "700",
            "marginRight": "5%",
            "textAlign": "end",
            "width": "50%",
          }
        }
      >
        NEW DEATHS
        <span>
          115
        </span>
      </p>
    </div>
    <div
      className=" card recovery"
    >
      <img
        alt="back icone"
        src="fleche.png"
        style={
          Object {
            "margin": "2.5% 5% 0 0",
          }
        }
      />
      <p
        style={
          Object {
            "display": "flex",
            "flexDirection": "column",
            "fontSize": "18px",
            "fontWeight": "700",
            "marginRight": "5%",
            "textAlign": "end",
            "width": "50%",
          }
        }
      >
        NEW RECOVERED
        <span>
          30
        </span>
      </p>
    </div>
    <div
      className=" card open-cases"
    >
      <img
        alt="back icone"
        src="fleche.png"
        style={
          Object {
            "margin": "2.5% 5% 0 0",
          }
        }
      />
      <p
        style={
          Object {
            "display": "flex",
            "flexDirection": "column",
            "fontSize": "18px",
            "fontWeight": "700",
            "marginRight": "5%",
            "textAlign": "end",
            "width": "50%",
          }
        }
      >
        NEW OPEN CASES
        <span>
          15
        </span>
      </p>
    </div>
  </div>
</div>
`;

exports[`render correctly navbar 1`] = `
<header
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgb(226, 77, 120)",
      "color": "#fff",
      "display": "flex",
      "fontFamily": "Lato, sans-serif",
      "height": "5rem",
      "margin": "0",
    }
  }
>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
      }
    }
  >
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      <img
        alt="back"
        src="back.png"
        style={
          Object {
            "cursor": "pointer",
          }
        }
      />
    </a>
    <span
      style={
        Object {
          "fontWeight": "700",
        }
      }
    >
      2021-12-20
    </span>
  </div>
  <span
    style={
      Object {
        "color": "#fff",
        "margin": "0 12.5%",
      }
    }
  >
    Tunisia national picture
  </span>
  <img
    alt="microphone"
    src="microphone.png"
    style={
      Object {
        "marginRight": "7.5%",
        "width": "5%",
      }
    }
  />
  <img
    alt="setting"
    src="settings.png"
    style={
      Object {
        "marginRight": "2.5%",
        "width": "5%",
      }
    }
  />
</header>
`;
